cmd/cancel.go

```diff
@@ -53,6 +54,8 @@ type cookedCancelCmdArgs struct {
 // dispatches the cancel Job order to the storage engine
 func (cca cookedCancelCmdArgs) process() error {
 	var cancelJobResponse common.CancelPauseResumeResponse
+	// White Glove: raise cancellation Telemetry event
+	telemetry.RaiseCancellationEvent(cca.jobID.String())
 	Rpc(common.ERpcCmd.CancelJob(), cca.jobID, &cancelJobResponse)
```

Comment:
Note that the cancel command somewhat fell out of use and into deprecation due to an overall structure change, and that this occurs in a separate process.

Reply:
@adreed-msft  - there is indeed less need for a _commmand_ to cancel a job; but all the internal cancellation still gets funnelled through this rountine.  So I'll mark this conversation resolved.